php/src/Google/Protobuf/RepeatedField.php

```diff
@@ -0,0 +1,252 @@
+<?php
+
+// Protocol Buffers - Google's data interchange format
+// Copyright 2008 Google Inc.  All rights reserved.
```

Comment:
Nit: do we even need this header comment? If so, should it be this year's date?

Reply:
Good question - I didn't update it since the file was essentially moved, but it did change a little... so maybe we should update it? I'm happy to do so if you'd like!